logstash-core/lib/logstash/agent.rb

```diff
@@ -480,7 +480,7 @@ def start_pipeline(id)
       begin
         pipeline.run
       rescue => e
-        @logger.error("Pipeline aborted due to error", :exception => e, :backtrace => e.backtrace)
+        @logger.error("Pipeline aborted due to error", :exception => e.class.name, :backtrace => e.backtrace)
```

Comment:
This is somewhat common to use this in the plugin, I've seen it a lot and I have use it myself.